sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -747,23 +747,25 @@ public HttpPipeline getHttpPipeline() {
     /**
      * Creates one or many models.
      * @param models The list of models to create. Each string corresponds to exactly one model.
-     * @return A {@link PagedFlux} of created models and the http response.
+     * @return A List of created models.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public PagedFlux<ModelData> createModels(List<String> models) {
-        return new PagedFlux<>(
-            () -> withContext(context -> createModelsSinglePageAsync(models, context)),
-            nextLink -> withContext(context -> Mono.empty()));
+    public Mono<List<ModelData>> createModels(List<String> models) {
```

Comment:
Nice! A list for create response is so much better than a pageable!

Reply:
yes, Srikanta wasn't the happiest about this decision, but he agreed that for the time being if the service team is not planning to start paginating the response of this request, this is the only way going forward.